smart-contracts/osmosis/contracts/cl-vault/src/helpers/getters.rs

```diff
@@ -43,20 +43,43 @@ pub fn get_asset0_value(
     Ok(total)
 }
 
-pub fn get_twap_price(
+pub fn get_position_balance(
```

Comment:
The function name does not match the implementation. Maybe rename it to get_asset_ratios or sth. in that direction?

Reply:
Addressed